signer/migrations/0017__remove_stacks_transactions_table.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE sbtc_signer.stacks_transactions CASCADE;
```

Comment:
On 2nd thought, could we get rid of `CASCADE`? Cascading deletes/drops are generally scary... Better to be explicit in removing their dependencies IMO, and it makes it clearer for our signer partners that manually review/apply their migrations.

Reply:
Done in https://github.com/stacks-sbtc/sbtc/pull/1601/commits/32285c2c8626cfc69ef8c84deca8123d4ed0a977